.gitlab-ci.yml

```diff
@@ -42,3 +43,41 @@ publish:image:mender:
 publish:image:saas:
   rules:
     - when: never
+
+#
+# Mender Helm Rolling release
+#
+.trigger:mender-helm-version-bump:
+  stage: version-bump
+  allow_failure: true
```

Comment:
Maybe this should go into the staging job to make it explicitly fail on production tags?

Reply:
This is an explicit workaround to a know issue: when you merge a dependabot pr, the `mender-test-bot` user has no permission to trigger a remote pipeline. So far I still don't know how to proceed (`mender-test-bot` has full permissions), so I want to make sure the service pipeline is not broken.
Then, when we'll solve the issue, we could remove the workaround for both prod and staging